.gitignore

```diff
@@ -4,3 +4,4 @@ boot/
 .classpath
 .project
 .scala_dependencies
+.idea/
```

Comment:
This should go to global gitignore - https://help.github.com/articles/ignoring-files/